code/_globalvars/lists/maintenance_loot.dm

```diff
@@ -240,13 +243,71 @@ GLOBAL_LIST_INIT(uncommon_loot, list(//uncommon: useful items
 		) = 1,
 ))
 
+
+
+GLOBAL_LIST_INIT(rarity_loot, list(//rare: really good items
+	list(//tools
+		/obj/item/weldingtool/hugetank = 1,
+		/obj/item/kitchen/knife = 1,
+		/obj/item/restraints/handcuffs = 1,
+		/obj/item/shield/riot/buckler = 1,
+		/obj/item/melee/baton/cattleprod/teleprod = 1,
+		/obj/item/throwing_star = 1,
+		/obj/item/pen/survival = 1,
+		/obj/item/flashlight/flashdark = 1,
+		/obj/item/assembly/flash/memorizer = 1,
+		) = 1,
+
+	list(//equipment
+		/obj/item/clothing/glasses/hud/security = 1,
+		/obj/item/clothing/glasses/sunglasses = 1,
+		/obj/item/storage/belt/security = 1,
+		/obj/item/storage/belt/military/assault = 1,
+		/obj/item/clothing/gloves/color/yellow = 1,
+		/obj/item/clothing/gloves/color/black = 1,
+		/obj/item/clothing/gloves/tackler/combat = 1,
+		/obj/item/clothing/head/helmet/justice = 1,
+		) = 1,
+
+	list(//paint
+		/obj/item/paint/red = 1,
+		/obj/item/paint/green = 1,
+		/obj/item/paint/blue = 1,
+		/obj/item/paint/yellow = 1,
+		/obj/item/paint/violet = 1,
+		/obj/item/paint/black = 1,
+		/obj/item/paint/white = 1,
+		/obj/item/paint/anycolor = 1,
+		/obj/item/paint/paint_remover = 1,
+		) = 1,
+
+	list(//medical and chemicals
+		list(//medkits
+			/obj/item/storage/box/hug/medical = 1,
+			/obj/item/storage/firstaid/regular = 1,
+			/obj/item/storage/firstaid/emergency = 1,
+			) = 1,
+		list(//medical chems
+			/obj/item/reagent_containers/hypospray/medipen/salacid = 1,
+			/obj/item/reagent_containers/hypospray/medipen/oxandrolone = 1,
+			/obj/item/reagent_containers/syringe/contraband/methamphetamine = 1,
+			) = 1,
+		) = 1,
+
+	list(//misc
+		/obj/item/disk/nuclear/fake = 1,
```

Comment:
Can this be deconned for syndicate tech? But also ❤️

Reply:
no i removed that like a year ago i think